internal/database/mariadb/component_instance_test.go

```diff
@@ -576,6 +578,44 @@ var _ = Describe("ComponentInstance - ", Label("database", "ComponentInstance"),
 					Expect(ci[0].Count).To(BeEquivalentTo(componentInstance.Count))
 					Expect(ci[0].ComponentVersionId).To(BeEquivalentTo(componentInstance.ComponentVersionId))
 					Expect(ci[0].ServiceId).To(BeEquivalentTo(componentInstance.ServiceId))
+					Expect(ci[0].ParentId).To(BeEquivalentTo(componentInstance.ParentId))
+				})
+			})
+			It("can insert correctly (without ParentID)", func() {
+				componentInstance, err := db.CreateComponentInstance(&newComponentInstance)
+
+				By("throwing no error", func() {
+					Expect(err).To(BeNil())
+				})
+				By("sets componentInstance id", func() {
+					Expect(componentInstance).NotTo(BeEquivalentTo(0))
+				})
+
+				componentInstanceFilter := &entity.ComponentInstanceFilter{
+					Id: []*int64{&componentInstance.Id},
+				}
+
+				ci, err := db.GetComponentInstances(componentInstanceFilter, nil)
```

Comment:
Why is GetComponentInstances called? The CreateComponentInstance function returns the created ComponentInstance. Same in the test with ParentID